pkg/liqonet/utils_test.go

```diff
@@ -0,0 +1,124 @@
+package liqonet_test
+
+import (
+	"context"
+	"time"
+
+	"github.com/liqotech/liqo/pkg/liqonet"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+var _ = Describe("Liqonet", func() {
+
+	Describe("Getting ClusterID info from ConfigMap", func() {
+
+		var (
+			ns             string
+			clusterIDKey   string
+			clusterIDValue string
+			configMapData  map[string]string
+			configMap      corev1.ConfigMap
+			backoff        wait.Backoff
+		)
+
+		BeforeEach(func() {
+			ns = "default"
+			clusterIDKey = "cluster-id"
+			clusterIDValue = "fake-cluster-id"
+			configMapData = make(map[string]string, 1)
+			configMapData[clusterIDKey] = clusterIDValue
+			configMap = corev1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "ConfigMap",
+					APIVersion: "v1",
+				},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      clusterIDKey,
+					Namespace: ns,
+				},
+				Data: configMapData,
+			}
+
+			// 3 attempts with 15 seconds sleep between one another
+			// for a total of 30 seconds
+			backoff = wait.Backoff{
```

Comment:
Since we just want to test the correctness of the function, I would use a backoff time way lower than 3*15 seconds.